setup.cfg

```diff
@@ -19,7 +19,7 @@ install_requires =
     numpy>=1.24
     scipy>=1.3
     astropy>=5.1
-    gwcs>=0.18
+    gwcs @ git+https://github.com/nden/gwcs.git@inverse-bbox
```

Comment:
Is this a PR on gwcs?

Reply:
This is what I was asking about above as well.

Reply:
Oh I think it is https://github.com/spacetelescope/gwcs/pull/498 that is already merged